models/audience.js

```diff
@@ -38,6 +40,28 @@ class Audience {
       }, {})
     }, isNil)
   }
+
+  get acsCsv () {
+    const csvStr = ACSMapping.join(",") + "\n";
+    const jsonData =  omitBy({
+      ...transform(
+        ACSMapping,
+        (result, path) => {
+          const value = get(this.message, path, undefined);
+          result[path] = value;
+        },
+      {})
+    }, isNil)
+    const jsonDataToCsvStr = ACSMapping.reduce((result, key, index) => {
+      return index === 0
+        ? `${result} ${jsonData[key]}`
+        : index === ACSMapping.length - 1
+        ? `${result}, ${jsonData[key]} \n`
+        : `${result}, ${jsonData[key]}`;
+    }, csvStr)
```

Comment:
This seems a bit complex right now for just two fields to check, but I did this with the intention of it growing in the future.